lecture/views.py

```
@@ -86,20 +86,68 @@ def lecture_pdf(request: HttpRequest,slug:str,pdf_id: int) -> HttpResponse:
+@require_http_methods(["GET", "POST"])
+def create_content(request: HttpRequest,course_slug:str,lecture_slug:str) -> HttpResponse:
+    if request.method == "POST":
+        form = LecturePDFForm(request.POST,request.FILES)  # Pass the user if needed
+        lecture = get_object_or_404(Lecture, slug=lecture_slug)
+        if form.is_valid():
+            saved_content = form.save(commit=False)
+            saved_content.lecture=lecture
+            saved_content.save()
+            messages.success(request, "Lecture Content created successfully!")
+            return redirect("lecture_home", slug=course_slug)  # Redirect to a relevant page
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)